### PR TITLE
add markdown from notion

### PR DESCRIPTION
**Pull request by ludralph**
This PR is to update the Readme.md file and remove references to our internal notion pages and adding subpages.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -19,10 +19,82 @@ git clone git@github.com:Jolg42/prisma-workshop.git
...
+
+Vous commencerez par configurer Prisma avec une base de données SQLite, vous apprendrez à modéliser des données avec Prisma et à effectuer des migrations de bases de données (*leçon 1*).
+
+Ensuite, vous découvrirez Prisma Client, un *query builder* *type-safe* qui peut être utilisé pour interroger votre base de données. Vous allez explorer différentes requêtes, du simple CRUD aux requêtes de relation, en passant par les filtres et la pagination (*leçon 2*).
+
+Ensuite, vous apprendrez comment utiliser Prisma Client pour implémenter les routes d'une API REST (*leçon 3*).
+
+Enfin, nous verrons comment utiliser Prisma Client pour implémenter les résolveurs d'une API GraphQL (*leçon 4*).
+
+# Agenda
```
Comment (nurul3101) [inline README.md]: We can remove the Agenda section as it's no longer needed

File: exploration-du-prisma-client.md [hunk 2]
```diff
@@ -0,0 +1,431 @@
...
+    }
+    
+    main()
+      .catch((e) => console.error(e))
+      .finally(async () => await prisma.disconnect())
+    ```
+    
+    - Ouvrez pour une capture d'écran de l'autocomplétion
+        
+        ![2%20Exploration%20du%20Prisma%20Client%20d15e11af643f4652b45ee9170e508689/Untitled.png](2%20Exploration%20du%20Prisma%20Client%20d15e11af643f4652b45ee9170e508689/Untitled.png)
```
Comment (nurul3101) [inline exploration-du-prisma-client.md]: This should be a link to image, right?

File: rest-api.md [hunk 3]
```diff
@@ -0,0 +1,263 @@
...
+
+Vous pouvez trouver les tâches pour cette leçon à l'intérieur du fichier `src/index.ts` marqué avec `TODO`. Votre objectif est d'insérer les bonnes requêtes Prisma Client pour chaque route de l'API REST.
+
+Notez que ceci n'est *pas* une leçon sur la conception d'API et que vous devriez réfléchir plus attentivement à la conception de vos opérations d'API dans une application.
+
+Si vous utilisez VS Code, vous pouvez installer l'extension [REST Client](https://marketplace.visualstudio.com/items?itemName=humao.rest-client) et tester votre implémentation en utilisant les appels HTTP fournis dans [`test.http`](https://github.com/nikolasburk/prisma-workshop/blob/rest-api/test.http).
+
+- Voir une démo rapide de l'extension VS Code REST Client
+    
+    ![3%20REST%20API%204973ea32555049049a2a5e24a026e768/rest-client.gif](3%20REST%20API%204973ea32555049049a2a5e24a026e768/rest-client.gif)
```
Comment (nurul3101) [inline rest-api.md]: Same for here, I think we need to manually upload the images/gifs in this repo and link to it

File: README.md (hunk 1 above)
Comment (ludralph) [reply README.md]: I have updated this

File: exploration-du-prisma-client.md (hunk 2 above)
Comment (ludralph) [reply exploration-du-prisma-client.md]: I have uploaded and linked to the image

File: rest-api.md (hunk 3 above)
Comment (ludralph) [reply rest-api.md]: I have uploaded and linked to the image

**Pull request merged**